src/components/MultipleSelect/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useMemo, useState } from 'react';
+import { TaskStatus } from 'utils/types';
+import './index.scss';
+
+interface MultipleSelectProps<T> {
+   options: T[];
+   listSeleted: T[];
+   onChange: (item: T[]) => void;
+   typeOfProp: any;
+   title: string;
+}
+
+function MultipleSelect({
+   options,
+   listSeleted,
+   onChange,
+   typeOfProp,
+   title,
+}: MultipleSelectProps<typeof typeOfProp>) {
+   const [show, setShow] = useState<boolean>(false);
+   options = useMemo<typeof typeOfProp>(() => {
+      if (typeOfProp === TaskStatus) {
+         return Array.from(options, (item) => TaskStatus[item]);
+      }
+   }, [typeOfProp, options]);
+
+   const handleSelectItem = (item: TaskStatus) => {
+      const _listSelected = [...listSeleted];
+      if (_listSelected.includes(item)) {
+         onChange(_listSelected.filter((value) => value !== item));
+      }
+      _listSelected.push(item);
+      onChange(_listSelected);
```

Comment:
- Chỗ này e fix chưa test lại đúng k :D, khi a chọn 1 thằng xong chọn nó tiếp thì vẫn được. Phải dùng else chứ e
- Có thể viết gọn lại, k cần phải clone ra 1 mảng trước

```suggestion
      if (listSelected.includes(item)) {
         handleRemoveItem(item);
      } else {
        onChange([
           ...listSelected,
           item
        ]);
      }
```